### DISCUSSION TITLE
Gaussian blur for image.

### DISCUSSION THREAD

**Discussion by JiuLing-zhang** [CommunityToolkit/Maui#407] [New Feature Discussions]
Is it possible to add the function of Gaussian blur for the image?

**Comment by GreenShadeZhang** [comment]
I think it's a very necessary feature to simply achieve some interesting effects.

**Reply by VladislavAntonyuk** [reply]
Could you please share a sample image?

**Reply by GreenShadeZhang** [reply]
Before
![图片](https://user-images.githubusercontent.com/24240675/177478127-2b14e138-4907-4e3b-8f3c-3dc388c50616.png)
After
![图片](https://user-images.githubusercontent.com/24240675/177478153-6cd9890e-b9cc-4aa4-a324-6370b1fb5dba.png)

Reference documentation

https://docs.microsoft.com/en-us/windows/win32/direct2d/gaussian-blur

**Reply by VladislavAntonyuk** [reply]
https://vladislavantonyuk.azurewebsites.net/articles/Creating-beautiful-image-effects-using-.NET-MAUI

**Reply by GreenShadeZhang** [reply]
> https://vladislavantonyuk.azurewebsites.net/articles/Creating-beautiful-image-effects-using-.NET-MAUI
Thank you very much for your realization, fantastic, I will refer to it well. I found another problem when I implemented it in other ways before, can you help see what the cause is? Below is a link to the question.
https://github.com/dotnet/maui/issues/8632

**Comment by CliffAgius** [comment]
Closed as answered
